services/bigpanda/config.go

```diff
@@ -29,17 +29,20 @@ type Config struct {
 	// Whether to skip the tls verification of the alerta host
 	InsecureSkipVerify bool `toml:"insecure-skip-verify" override:"insecure-skip-verify"`
 
-	//Optional alert api URL, if not specified https://api.bigpanda.io/data/v2/alerts is used
+	// BigPanda Alert webhook URL,
+	// https://api.bigpanda.io/data/v2/alerts
 	URL string `toml:"url" override:"url"`
 }
 
 func NewConfig() Config {
-	return Config{
-		URL: defaultBigPandaAlertApi,
-	}
+	return Config{}
```

Comment:
Why this change?

Reply:
I think that the dafault url https://api.bigpanda.io/data/v2/alerts should not be hardcoded. For now we use the generic REST API, but we want BigPanda to add our integration among the supported official integrations. Instead of a generic REST API url, we will get a special webhook url like https://inbound.bigpanda.io/influxdata/alerts, which will be generated in the bigpanda admin console.

Reply:
Reasonable defaults are a good idea for users, this was a specific ask, for a previous PR.